package.json

```diff
@@ -76,5 +77,8 @@
     "lint": "standard",
     "test": "echo 'not yet implemented, wanna help?'"
   },
+  "standard": {
+    "parser": "babel-eslint"
```

Comment:
what about standard then? You use it here but is not listed as devDependecies either 😕